Remotion/Validation/UnitTests/Validators/NotEmptyValidatorTest.cs

```diff
@@ -77,8 +76,8 @@ public void Validate_WithPropertyValueIsEmptyCollection_ReturnsSingleValidationF
     [Test]
     public void Validate_WithPropertyValueIsNonEmptyCollection_ReturnsNoValidationFailures ()
```

Comment:
Please check, there's probably a couple of tests that are not applicable here or must be added for byte[]. And when we split string and byte[], some tests get moved to just one of the testfixtures.